[2024-08-24T12:20:07.105+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-24T12:20:07.119+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: spark_airflow_dag.spark_submit_task manual__2024-08-24T11:34:17.052163+00:00 [queued]>
[2024-08-24T12:20:07.126+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: spark_airflow_dag.spark_submit_task manual__2024-08-24T11:34:17.052163+00:00 [queued]>
[2024-08-24T12:20:07.127+0000] {taskinstance.py:2856} INFO - Starting attempt 6 of 6
[2024-08-24T12:20:07.140+0000] {taskinstance.py:2879} INFO - Executing <Task(SparkSubmitOperator): spark_submit_task> on 2024-08-24 11:34:17.052163+00:00
[2024-08-24T12:20:07.146+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2587) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-24T12:20:07.148+0000] {standard_task_runner.py:72} INFO - Started process 2589 to run task
[2024-08-24T12:20:07.148+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'spark_***_dag', 'spark_submit_task', 'manual__2024-08-24T11:34:17.052163+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/spark_jobs_dag.py', '--cfg-path', '/tmp/tmpr4wu6tm7']
[2024-08-24T12:20:07.148+0000] {standard_task_runner.py:105} INFO - Job 68: Subtask spark_submit_task
[2024-08-24T12:20:07.160+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-08-24T12:20:07.187+0000] {task_command.py:467} INFO - Running <TaskInstance: spark_airflow_dag.spark_submit_task manual__2024-08-24T11:34:17.052163+00:00 [running]> on host 62929844d775
[2024-08-24T12:20:07.267+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='spark_***_dag' AIRFLOW_CTX_TASK_ID='spark_submit_task' AIRFLOW_CTX_EXECUTION_DATE='2024-08-24T11:34:17.052163+00:00' AIRFLOW_CTX_TRY_NUMBER='6' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-24T11:34:17.052163+00:00'
[2024-08-24T12:20:07.268+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-08-24T12:20:07.268+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-08-24T12:20:07.268+0000] {logging_mixin.py:190} INFO - Current task name:spark_submit_task state:running start_date:2024-08-24 12:20:07.120263+00:00
[2024-08-24T12:20:07.268+0000] {logging_mixin.py:190} INFO - Dag name:spark_***_dag and current dag run status:running
[2024-08-24T12:20:07.269+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-24T12:20:07.277+0000] {base.py:84} INFO - Retrieving connection 'spark_local'
[2024-08-24T12:20:07.278+0000] {spark_submit.py:403} INFO - Spark-Submit cmd: spark-submit --master spark://543daafc0f5f:7077 --name arrow-spark --queue root.default --deploy-mode client /opt/***/dags/spark_etl_script_docker.py
[2024-08-24T12:20:10.390+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO SparkContext: Running Spark version 3.5.2
[2024-08-24T12:20:10.392+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO SparkContext: OS info Linux, 5.15.153.1-microsoft-standard-WSL2, amd64
[2024-08-24T12:20:10.392+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO SparkContext: Java version 17.0.12
[2024-08-24T12:20:10.440+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-08-24T12:20:10.524+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO ResourceUtils: ==============================================================
[2024-08-24T12:20:10.525+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-08-24T12:20:10.526+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO ResourceUtils: ==============================================================
[2024-08-24T12:20:10.527+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO SparkContext: Submitted application: DataExtraction
[2024-08-24T12:20:10.548+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-08-24T12:20:10.556+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO ResourceProfile: Limiting resource is cpu
[2024-08-24T12:20:10.557+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-08-24T12:20:10.609+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO SecurityManager: Changing view acls to: default
[2024-08-24T12:20:10.610+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO SecurityManager: Changing modify acls to: default
[2024-08-24T12:20:10.611+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO SecurityManager: Changing view acls groups to:
[2024-08-24T12:20:10.612+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO SecurityManager: Changing modify acls groups to:
[2024-08-24T12:20:10.612+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: default; groups with view permissions: EMPTY; users with modify permissions: default; groups with modify permissions: EMPTY
[2024-08-24T12:20:10.864+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO Utils: Successfully started service 'sparkDriver' on port 35023.
[2024-08-24T12:20:10.900+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO SparkEnv: Registering MapOutputTracker
[2024-08-24T12:20:10.943+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO SparkEnv: Registering BlockManagerMaster
[2024-08-24T12:20:10.967+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-08-24T12:20:10.968+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-08-24T12:20:10.988+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:10 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-08-24T12:20:11.013+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-09459c9b-bb2f-4878-8c71-07298bac6477
[2024-08-24T12:20:11.030+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-08-24T12:20:11.047+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-08-24T12:20:11.148+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-08-24T12:20:11.201+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-08-24T12:20:11.306+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://543daafc0f5f:7077...
[2024-08-24T12:20:11.346+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO TransportClientFactory: Successfully created connection to 543daafc0f5f/172.19.0.2:7077 after 23 ms (0 ms spent in bootstraps)
[2024-08-24T12:20:11.495+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20240824122011-0000
[2024-08-24T12:20:11.515+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 46237.
[2024-08-24T12:20:11.515+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO NettyBlockTransferService: Server created on 62929844d775:46237
[2024-08-24T12:20:11.521+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-08-24T12:20:11.536+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 62929844d775, 46237, None)
[2024-08-24T12:20:11.543+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO BlockManagerMasterEndpoint: Registering block manager 62929844d775:46237 with 434.4 MiB RAM, BlockManagerId(driver, 62929844d775, 46237, None)
[2024-08-24T12:20:11.547+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 62929844d775, 46237, None)
[2024-08-24T12:20:11.550+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 62929844d775, 46237, None)
[2024-08-24T12:20:11.573+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20240824122011-0000/0 on worker-20240824121709-172.19.0.4-42027 (172.19.0.4:42027) with 1 core(s)
[2024-08-24T12:20:11.577+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO StandaloneSchedulerBackend: Granted executor ID app-20240824122011-0000/0 on hostPort 172.19.0.4:42027 with 1 core(s), 1024.0 MiB RAM
[2024-08-24T12:20:11.788+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20240824122011-0000/0 is now RUNNING
[2024-08-24T12:20:11.902+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:11 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2024-08-24T12:20:12.218+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:12 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2024-08-24T12:20:12.225+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:12 INFO SharedState: Warehouse path is 'file:/opt/***/spark-warehouse'.
[2024-08-24T12:20:13.329+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:13 INFO InMemoryFileIndex: It took 59 ms to list leaf files for 1 paths.
[2024-08-24T12:20:13.419+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:13 INFO InMemoryFileIndex: It took 3 ms to list leaf files for 1 paths.
[2024-08-24T12:20:14.681+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:14 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Registered executor NettyRpcEndpointRef(spark-client://Executor) (172.19.0.4:54994) with ID 0,  ResourceProfileId 0
[2024-08-24T12:20:14.773+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:14 INFO BlockManagerMasterEndpoint: Registering block manager 172.19.0.4:33101 with 434.4 MiB RAM, BlockManagerId(0, 172.19.0.4, 33101, None)
[2024-08-24T12:20:16.077+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO FileSourceStrategy: Pushed Filters:
[2024-08-24T12:20:16.079+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO FileSourceStrategy: Post-Scan Filters: (length(trim(value#0, None)) > 0)
[2024-08-24T12:20:16.551+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO CodeGenerator: Code generated in 183.779488 ms
[2024-08-24T12:20:16.605+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO MemoryStore: Block broadcast_0 stored as values in memory (estimated size 199.3 KiB, free 434.2 MiB)
[2024-08-24T12:20:16.676+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO MemoryStore: Block broadcast_0_piece0 stored as bytes in memory (estimated size 34.2 KiB, free 434.2 MiB)
[2024-08-24T12:20:16.679+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO BlockManagerInfo: Added broadcast_0_piece0 in memory on 62929844d775:46237 (size: 34.2 KiB, free: 434.4 MiB)
[2024-08-24T12:20:16.683+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO SparkContext: Created broadcast 0 from csv at NativeMethodAccessorImpl.java:0
[2024-08-24T12:20:16.694+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO FileSourceScanExec: Planning scan with bin packing, max size: 4194304 bytes, open cost is considered as scanning 4194304 bytes.
[2024-08-24T12:20:16.772+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO SparkContext: Starting job: csv at NativeMethodAccessorImpl.java:0
[2024-08-24T12:20:16.783+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO DAGScheduler: Got job 0 (csv at NativeMethodAccessorImpl.java:0) with 1 output partitions
[2024-08-24T12:20:16.784+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO DAGScheduler: Final stage: ResultStage 0 (csv at NativeMethodAccessorImpl.java:0)
[2024-08-24T12:20:16.785+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO DAGScheduler: Parents of final stage: List()
[2024-08-24T12:20:16.785+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO DAGScheduler: Missing parents: List()
[2024-08-24T12:20:16.788+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO DAGScheduler: Submitting ResultStage 0 (MapPartitionsRDD[3] at csv at NativeMethodAccessorImpl.java:0), which has no missing parents
[2024-08-24T12:20:16.835+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO MemoryStore: Block broadcast_1 stored as values in memory (estimated size 13.5 KiB, free 434.2 MiB)
[2024-08-24T12:20:16.842+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO MemoryStore: Block broadcast_1_piece0 stored as bytes in memory (estimated size 6.4 KiB, free 434.2 MiB)
[2024-08-24T12:20:16.843+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO BlockManagerInfo: Added broadcast_1_piece0 in memory on 62929844d775:46237 (size: 6.4 KiB, free: 434.4 MiB)
[2024-08-24T12:20:16.843+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO SparkContext: Created broadcast 1 from broadcast at DAGScheduler.scala:1585
[2024-08-24T12:20:16.855+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO DAGScheduler: Submitting 1 missing tasks from ResultStage 0 (MapPartitionsRDD[3] at csv at NativeMethodAccessorImpl.java:0) (first 15 tasks are for partitions Vector(0))
[2024-08-24T12:20:16.856+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO TaskSchedulerImpl: Adding task set 0.0 with 1 tasks resource profile 0
[2024-08-24T12:20:16.876+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:16 INFO TaskSetManager: Starting task 0.0 in stage 0.0 (TID 0) (172.19.0.4, executor 0, partition 0, PROCESS_LOCAL, 9599 bytes)
[2024-08-24T12:20:17.060+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:17 INFO BlockManagerInfo: Added broadcast_1_piece0 in memory on 172.19.0.4:33101 (size: 6.4 KiB, free: 434.4 MiB)
[2024-08-24T12:20:17.784+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:17 INFO BlockManagerInfo: Added broadcast_0_piece0 in memory on 172.19.0.4:33101 (size: 34.2 KiB, free: 434.4 MiB)
[2024-08-24T12:20:18.283+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 WARN TaskSetManager: Lost task 0.0 in stage 0.0 (TID 0) (172.19.0.4 executor 0): org.apache.spark.SparkFileNotFoundException: File file:/opt/***/dataframe.csv does not exist
[2024-08-24T12:20:18.284+0000] {spark_submit.py:579} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2024-08-24T12:20:18.284+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:781)
[2024-08-24T12:20:18.284+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2024-08-24T12:20:18.284+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2024-08-24T12:20:18.284+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2024-08-24T12:20:18.285+0000] {spark_submit.py:579} INFO - at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2024-08-24T12:20:18.285+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage1.processNext(Unknown Source)
[2024-08-24T12:20:18.285+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2024-08-24T12:20:18.285+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2024-08-24T12:20:18.285+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2024-08-24T12:20:18.286+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-08-24T12:20:18.286+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-08-24T12:20:18.286+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-08-24T12:20:18.286+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-08-24T12:20:18.287+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-08-24T12:20:18.287+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-08-24T12:20:18.287+0000] {spark_submit.py:579} INFO - at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-08-24T12:20:18.287+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-08-24T12:20:18.287+0000] {spark_submit.py:579} INFO - at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-08-24T12:20:18.288+0000] {spark_submit.py:579} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-08-24T12:20:18.288+0000] {spark_submit.py:579} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-08-24T12:20:18.288+0000] {spark_submit.py:579} INFO - at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-08-24T12:20:18.288+0000] {spark_submit.py:579} INFO - at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-08-24T12:20:18.288+0000] {spark_submit.py:579} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2024-08-24T12:20:18.289+0000] {spark_submit.py:579} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2024-08-24T12:20:18.289+0000] {spark_submit.py:579} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-08-24T12:20:18.289+0000] {spark_submit.py:579} INFO - 
[2024-08-24T12:20:18.289+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO TaskSetManager: Starting task 0.1 in stage 0.0 (TID 1) (172.19.0.4, executor 0, partition 0, PROCESS_LOCAL, 9599 bytes)
[2024-08-24T12:20:18.323+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO TaskSetManager: Lost task 0.1 in stage 0.0 (TID 1) on 172.19.0.4, executor 0: org.apache.spark.SparkFileNotFoundException (File file:/opt/***/dataframe.csv does not exist
[2024-08-24T12:20:18.323+0000] {spark_submit.py:579} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 1]
[2024-08-24T12:20:18.324+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO TaskSetManager: Starting task 0.2 in stage 0.0 (TID 2) (172.19.0.4, executor 0, partition 0, PROCESS_LOCAL, 9599 bytes)
[2024-08-24T12:20:18.343+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO TaskSetManager: Lost task 0.2 in stage 0.0 (TID 2) on 172.19.0.4, executor 0: org.apache.spark.SparkFileNotFoundException (File file:/opt/***/dataframe.csv does not exist
[2024-08-24T12:20:18.344+0000] {spark_submit.py:579} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 2]
[2024-08-24T12:20:18.344+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO TaskSetManager: Starting task 0.3 in stage 0.0 (TID 3) (172.19.0.4, executor 0, partition 0, PROCESS_LOCAL, 9599 bytes)
[2024-08-24T12:20:18.364+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO TaskSetManager: Lost task 0.3 in stage 0.0 (TID 3) on 172.19.0.4, executor 0: org.apache.spark.SparkFileNotFoundException (File file:/opt/***/dataframe.csv does not exist
[2024-08-24T12:20:18.365+0000] {spark_submit.py:579} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.) [duplicate 3]
[2024-08-24T12:20:18.365+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 ERROR TaskSetManager: Task 0 in stage 0.0 failed 4 times; aborting job
[2024-08-24T12:20:18.368+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO TaskSchedulerImpl: Removed TaskSet 0.0, whose tasks have all completed, from pool
[2024-08-24T12:20:18.369+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO TaskSchedulerImpl: Cancelling stage 0
[2024-08-24T12:20:18.369+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO TaskSchedulerImpl: Killing all running tasks in stage 0: Stage cancelled: Job aborted due to stage failure: Task 0 in stage 0.0 failed 4 times, most recent failure: Lost task 0.3 in stage 0.0 (TID 3) (172.19.0.4 executor 0): org.apache.spark.SparkFileNotFoundException: File file:/opt/***/dataframe.csv does not exist
[2024-08-24T12:20:18.369+0000] {spark_submit.py:579} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2024-08-24T12:20:18.369+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:781)
[2024-08-24T12:20:18.369+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2024-08-24T12:20:18.370+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2024-08-24T12:20:18.370+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2024-08-24T12:20:18.370+0000] {spark_submit.py:579} INFO - at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2024-08-24T12:20:18.370+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage1.processNext(Unknown Source)
[2024-08-24T12:20:18.370+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2024-08-24T12:20:18.370+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2024-08-24T12:20:18.371+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2024-08-24T12:20:18.371+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-08-24T12:20:18.371+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-08-24T12:20:18.371+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-08-24T12:20:18.371+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-08-24T12:20:18.371+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-08-24T12:20:18.372+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-08-24T12:20:18.372+0000] {spark_submit.py:579} INFO - at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-08-24T12:20:18.372+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-08-24T12:20:18.372+0000] {spark_submit.py:579} INFO - at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-08-24T12:20:18.372+0000] {spark_submit.py:579} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-08-24T12:20:18.372+0000] {spark_submit.py:579} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-08-24T12:20:18.373+0000] {spark_submit.py:579} INFO - at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-08-24T12:20:18.373+0000] {spark_submit.py:579} INFO - at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-08-24T12:20:18.373+0000] {spark_submit.py:579} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2024-08-24T12:20:18.373+0000] {spark_submit.py:579} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2024-08-24T12:20:18.373+0000] {spark_submit.py:579} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-08-24T12:20:18.373+0000] {spark_submit.py:579} INFO - 
[2024-08-24T12:20:18.374+0000] {spark_submit.py:579} INFO - Driver stacktrace:
[2024-08-24T12:20:18.374+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO DAGScheduler: ResultStage 0 (csv at NativeMethodAccessorImpl.java:0) failed in 1.573 s due to Job aborted due to stage failure: Task 0 in stage 0.0 failed 4 times, most recent failure: Lost task 0.3 in stage 0.0 (TID 3) (172.19.0.4 executor 0): org.apache.spark.SparkFileNotFoundException: File file:/opt/***/dataframe.csv does not exist
[2024-08-24T12:20:18.374+0000] {spark_submit.py:579} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2024-08-24T12:20:18.374+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:781)
[2024-08-24T12:20:18.374+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2024-08-24T12:20:18.375+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2024-08-24T12:20:18.375+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2024-08-24T12:20:18.375+0000] {spark_submit.py:579} INFO - at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2024-08-24T12:20:18.375+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage1.processNext(Unknown Source)
[2024-08-24T12:20:18.375+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2024-08-24T12:20:18.375+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2024-08-24T12:20:18.375+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2024-08-24T12:20:18.376+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-08-24T12:20:18.376+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-08-24T12:20:18.376+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-08-24T12:20:18.376+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-08-24T12:20:18.376+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-08-24T12:20:18.376+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-08-24T12:20:18.377+0000] {spark_submit.py:579} INFO - at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-08-24T12:20:18.377+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-08-24T12:20:18.377+0000] {spark_submit.py:579} INFO - at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-08-24T12:20:18.377+0000] {spark_submit.py:579} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-08-24T12:20:18.377+0000] {spark_submit.py:579} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-08-24T12:20:18.377+0000] {spark_submit.py:579} INFO - at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-08-24T12:20:18.377+0000] {spark_submit.py:579} INFO - at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-08-24T12:20:18.378+0000] {spark_submit.py:579} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2024-08-24T12:20:18.378+0000] {spark_submit.py:579} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2024-08-24T12:20:18.378+0000] {spark_submit.py:579} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-08-24T12:20:18.378+0000] {spark_submit.py:579} INFO - 
[2024-08-24T12:20:18.378+0000] {spark_submit.py:579} INFO - Driver stacktrace:
[2024-08-24T12:20:18.378+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO DAGScheduler: Job 0 failed: csv at NativeMethodAccessorImpl.java:0, took 1.600205 s
[2024-08-24T12:20:18.551+0000] {spark_submit.py:579} INFO - Traceback (most recent call last):
[2024-08-24T12:20:18.551+0000] {spark_submit.py:579} INFO - File "/opt/***/dags/spark_etl_script_docker.py", line 22, in <module>
[2024-08-24T12:20:18.552+0000] {spark_submit.py:579} INFO - spark.read.csv("dataframe.csv").show(100, False)
[2024-08-24T12:20:18.552+0000] {spark_submit.py:579} INFO - ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-08-24T12:20:18.552+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/readwriter.py", line 740, in csv
[2024-08-24T12:20:18.552+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1322, in __call__
[2024-08-24T12:20:18.552+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/errors/exceptions/captured.py", line 179, in deco
[2024-08-24T12:20:18.553+0000] {spark_submit.py:579} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/protocol.py", line 326, in get_return_value
[2024-08-24T12:20:18.599+0000] {spark_submit.py:579} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling o24.csv.
[2024-08-24T12:20:18.599+0000] {spark_submit.py:579} INFO - : org.apache.spark.SparkException: Job aborted due to stage failure: Task 0 in stage 0.0 failed 4 times, most recent failure: Lost task 0.3 in stage 0.0 (TID 3) (172.19.0.4 executor 0): org.apache.spark.SparkFileNotFoundException: File file:/opt/***/dataframe.csv does not exist
[2024-08-24T12:20:18.600+0000] {spark_submit.py:579} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2024-08-24T12:20:18.600+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:781)
[2024-08-24T12:20:18.600+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2024-08-24T12:20:18.600+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2024-08-24T12:20:18.601+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2024-08-24T12:20:18.601+0000] {spark_submit.py:579} INFO - at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2024-08-24T12:20:18.601+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage1.processNext(Unknown Source)
[2024-08-24T12:20:18.601+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2024-08-24T12:20:18.601+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2024-08-24T12:20:18.602+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2024-08-24T12:20:18.602+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-08-24T12:20:18.602+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-08-24T12:20:18.602+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-08-24T12:20:18.602+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-08-24T12:20:18.602+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-08-24T12:20:18.603+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-08-24T12:20:18.603+0000] {spark_submit.py:579} INFO - at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-08-24T12:20:18.603+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-08-24T12:20:18.603+0000] {spark_submit.py:579} INFO - at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-08-24T12:20:18.603+0000] {spark_submit.py:579} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-08-24T12:20:18.604+0000] {spark_submit.py:579} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-08-24T12:20:18.604+0000] {spark_submit.py:579} INFO - at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-08-24T12:20:18.604+0000] {spark_submit.py:579} INFO - at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-08-24T12:20:18.604+0000] {spark_submit.py:579} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2024-08-24T12:20:18.604+0000] {spark_submit.py:579} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2024-08-24T12:20:18.605+0000] {spark_submit.py:579} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-08-24T12:20:18.605+0000] {spark_submit.py:579} INFO - 
[2024-08-24T12:20:18.605+0000] {spark_submit.py:579} INFO - Driver stacktrace:
[2024-08-24T12:20:18.606+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.DAGScheduler.failJobAndIndependentStages(DAGScheduler.scala:2856)
[2024-08-24T12:20:18.606+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.DAGScheduler.$anonfun$abortStage$2(DAGScheduler.scala:2792)
[2024-08-24T12:20:18.606+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.DAGScheduler.$anonfun$abortStage$2$adapted(DAGScheduler.scala:2791)
[2024-08-24T12:20:18.607+0000] {spark_submit.py:579} INFO - at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:62)
[2024-08-24T12:20:18.607+0000] {spark_submit.py:579} INFO - at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:55)
[2024-08-24T12:20:18.607+0000] {spark_submit.py:579} INFO - at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:49)
[2024-08-24T12:20:18.607+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.DAGScheduler.abortStage(DAGScheduler.scala:2791)
[2024-08-24T12:20:18.608+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.DAGScheduler.$anonfun$handleTaskSetFailed$1(DAGScheduler.scala:1247)
[2024-08-24T12:20:18.608+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.DAGScheduler.$anonfun$handleTaskSetFailed$1$adapted(DAGScheduler.scala:1247)
[2024-08-24T12:20:18.608+0000] {spark_submit.py:579} INFO - at scala.Option.foreach(Option.scala:407)
[2024-08-24T12:20:18.609+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.DAGScheduler.handleTaskSetFailed(DAGScheduler.scala:1247)
[2024-08-24T12:20:18.609+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.doOnReceive(DAGScheduler.scala:3060)
[2024-08-24T12:20:18.609+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:2994)
[2024-08-24T12:20:18.610+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.DAGSchedulerEventProcessLoop.onReceive(DAGScheduler.scala:2983)
[2024-08-24T12:20:18.610+0000] {spark_submit.py:579} INFO - at org.apache.spark.util.EventLoop$$anon$1.run(EventLoop.scala:49)
[2024-08-24T12:20:18.610+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.DAGScheduler.runJob(DAGScheduler.scala:989)
[2024-08-24T12:20:18.610+0000] {spark_submit.py:579} INFO - at org.apache.spark.SparkContext.runJob(SparkContext.scala:2393)
[2024-08-24T12:20:18.611+0000] {spark_submit.py:579} INFO - at org.apache.spark.SparkContext.runJob(SparkContext.scala:2414)
[2024-08-24T12:20:18.611+0000] {spark_submit.py:579} INFO - at org.apache.spark.SparkContext.runJob(SparkContext.scala:2433)
[2024-08-24T12:20:18.611+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.SparkPlan.executeTake(SparkPlan.scala:530)
[2024-08-24T12:20:18.611+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.SparkPlan.executeTake(SparkPlan.scala:483)
[2024-08-24T12:20:18.612+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.CollectLimitExec.executeCollect(limit.scala:61)
[2024-08-24T12:20:18.612+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.Dataset.collectFromPlan(Dataset.scala:4334)
[2024-08-24T12:20:18.612+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.Dataset.$anonfun$head$1(Dataset.scala:3316)
[2024-08-24T12:20:18.612+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.Dataset.$anonfun$withAction$2(Dataset.scala:4324)
[2024-08-24T12:20:18.613+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.QueryExecution$.withInternalError(QueryExecution.scala:546)
[2024-08-24T12:20:18.613+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.Dataset.$anonfun$withAction$1(Dataset.scala:4322)
[2024-08-24T12:20:18.613+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withNewExecutionId$6(SQLExecution.scala:125)
[2024-08-24T12:20:18.614+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.SQLExecution$.withSQLConfPropagated(SQLExecution.scala:201)
[2024-08-24T12:20:18.614+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.SQLExecution$.$anonfun$withNewExecutionId$1(SQLExecution.scala:108)
[2024-08-24T12:20:18.614+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.SparkSession.withActive(SparkSession.scala:900)
[2024-08-24T12:20:18.614+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.SQLExecution$.withNewExecutionId(SQLExecution.scala:66)
[2024-08-24T12:20:18.614+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.Dataset.withAction(Dataset.scala:4322)
[2024-08-24T12:20:18.615+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.Dataset.head(Dataset.scala:3316)
[2024-08-24T12:20:18.615+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.Dataset.take(Dataset.scala:3539)
[2024-08-24T12:20:18.615+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.csv.TextInputCSVDataSource$.infer(CSVDataSource.scala:111)
[2024-08-24T12:20:18.615+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.csv.CSVDataSource.inferSchema(CSVDataSource.scala:64)
[2024-08-24T12:20:18.615+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.csv.CSVFileFormat.inferSchema(CSVFileFormat.scala:62)
[2024-08-24T12:20:18.616+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.DataSource.$anonfun$getOrInferFileFormatSchema$11(DataSource.scala:208)
[2024-08-24T12:20:18.616+0000] {spark_submit.py:579} INFO - at scala.Option.orElse(Option.scala:447)
[2024-08-24T12:20:18.616+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.DataSource.getOrInferFileFormatSchema(DataSource.scala:205)
[2024-08-24T12:20:18.616+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.DataSource.resolveRelation(DataSource.scala:407)
[2024-08-24T12:20:18.616+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.DataFrameReader.loadV1Source(DataFrameReader.scala:229)
[2024-08-24T12:20:18.616+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.DataFrameReader.$anonfun$load$2(DataFrameReader.scala:211)
[2024-08-24T12:20:18.617+0000] {spark_submit.py:579} INFO - at scala.Option.getOrElse(Option.scala:189)
[2024-08-24T12:20:18.617+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.DataFrameReader.load(DataFrameReader.scala:211)
[2024-08-24T12:20:18.617+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.DataFrameReader.csv(DataFrameReader.scala:538)
[2024-08-24T12:20:18.617+0000] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-08-24T12:20:18.617+0000] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[2024-08-24T12:20:18.618+0000] {spark_submit.py:579} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-08-24T12:20:18.618+0000] {spark_submit.py:579} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:569)
[2024-08-24T12:20:18.618+0000] {spark_submit.py:579} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2024-08-24T12:20:18.618+0000] {spark_submit.py:579} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-08-24T12:20:18.618+0000] {spark_submit.py:579} INFO - at py4j.Gateway.invoke(Gateway.java:282)
[2024-08-24T12:20:18.618+0000] {spark_submit.py:579} INFO - at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2024-08-24T12:20:18.619+0000] {spark_submit.py:579} INFO - at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2024-08-24T12:20:18.619+0000] {spark_submit.py:579} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-08-24T12:20:18.619+0000] {spark_submit.py:579} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-08-24T12:20:18.619+0000] {spark_submit.py:579} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-08-24T12:20:18.620+0000] {spark_submit.py:579} INFO - Caused by: org.apache.spark.SparkFileNotFoundException: File file:/opt/***/dataframe.csv does not exist
[2024-08-24T12:20:18.620+0000] {spark_submit.py:579} INFO - It is possible the underlying files have been updated. You can explicitly invalidate the cache in Spark by running 'REFRESH TABLE tableName' command in SQL or by recreating the Dataset/DataFrame involved.
[2024-08-24T12:20:18.620+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.errors.QueryExecutionErrors$.readCurrentFileNotFoundError(QueryExecutionErrors.scala:781)
[2024-08-24T12:20:18.620+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.org$apache$spark$sql$execution$datasources$FileScanRDD$$anon$$readCurrentFile(FileScanRDD.scala:220)
[2024-08-24T12:20:18.621+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.nextIterator(FileScanRDD.scala:279)
[2024-08-24T12:20:18.621+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.datasources.FileScanRDD$$anon$1.hasNext(FileScanRDD.scala:129)
[2024-08-24T12:20:18.621+0000] {spark_submit.py:579} INFO - at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460)
[2024-08-24T12:20:18.621+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage1.processNext(Unknown Source)
[2024-08-24T12:20:18.622+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:43)
[2024-08-24T12:20:18.622+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43)
[2024-08-24T12:20:18.622+0000] {spark_submit.py:579} INFO - at org.apache.spark.sql.execution.SparkPlan.$anonfun$getByteArrayRdd$1(SparkPlan.scala:388)
[2024-08-24T12:20:18.622+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2(RDD.scala:893)
[2024-08-24T12:20:18.623+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.$anonfun$mapPartitionsInternal$2$adapted(RDD.scala:893)
[2024-08-24T12:20:18.623+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
[2024-08-24T12:20:18.623+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:367)
[2024-08-24T12:20:18.623+0000] {spark_submit.py:579} INFO - at org.apache.spark.rdd.RDD.iterator(RDD.scala:331)
[2024-08-24T12:20:18.623+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:93)
[2024-08-24T12:20:18.624+0000] {spark_submit.py:579} INFO - at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:166)
[2024-08-24T12:20:18.624+0000] {spark_submit.py:579} INFO - at org.apache.spark.scheduler.Task.run(Task.scala:141)
[2024-08-24T12:20:18.624+0000] {spark_submit.py:579} INFO - at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:620)
[2024-08-24T12:20:18.624+0000] {spark_submit.py:579} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64)
[2024-08-24T12:20:18.625+0000] {spark_submit.py:579} INFO - at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61)
[2024-08-24T12:20:18.625+0000] {spark_submit.py:579} INFO - at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:94)
[2024-08-24T12:20:18.625+0000] {spark_submit.py:579} INFO - at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:623)
[2024-08-24T12:20:18.625+0000] {spark_submit.py:579} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2024-08-24T12:20:18.626+0000] {spark_submit.py:579} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2024-08-24T12:20:18.626+0000] {spark_submit.py:579} INFO - ... 1 more
[2024-08-24T12:20:18.626+0000] {spark_submit.py:579} INFO - 
[2024-08-24T12:20:18.677+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO SparkContext: Invoking stop() from shutdown hook
[2024-08-24T12:20:18.677+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-08-24T12:20:18.686+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO SparkUI: Stopped Spark web UI at http://62929844d775:4040
[2024-08-24T12:20:18.690+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO StandaloneSchedulerBackend: Shutting down all executors
[2024-08-24T12:20:18.690+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Asking each executor to shut down
[2024-08-24T12:20:18.706+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-08-24T12:20:18.725+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO MemoryStore: MemoryStore cleared
[2024-08-24T12:20:18.725+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO BlockManager: BlockManager stopped
[2024-08-24T12:20:18.730+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-08-24T12:20:18.732+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-08-24T12:20:18.742+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO SparkContext: Successfully stopped SparkContext
[2024-08-24T12:20:18.743+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO ShutdownHookManager: Shutdown hook called
[2024-08-24T12:20:18.743+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO ShutdownHookManager: Deleting directory /tmp/spark-eab97fe2-366a-4aab-91e5-d8103b190a55
[2024-08-24T12:20:18.748+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO ShutdownHookManager: Deleting directory /tmp/spark-5371d85d-c633-4fa1-9bfd-889c74f3d6b7
[2024-08-24T12:20:18.752+0000] {spark_submit.py:579} INFO - 24/08/24 12:20:18 INFO ShutdownHookManager: Deleting directory /tmp/spark-5371d85d-c633-4fa1-9bfd-889c74f3d6b7/pyspark-dbe74d0d-d46a-4e2a-b37c-e169ae3d08f4
[2024-08-24T12:20:18.794+0000] {taskinstance.py:3301} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 505, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://543daafc0f5f:7077 --name arrow-spark --queue root.default --deploy-mode client /opt/***/dags/spark_etl_script_docker.py. Error code is: 1.
[2024-08-24T12:20:18.798+0000] {logging_mixin.py:190} INFO - Task instance in failure state
[2024-08-24T12:20:18.798+0000] {logging_mixin.py:190} INFO - Task start:2024-08-24 12:20:07.120263+00:00 end:2024-08-24 12:20:18.797569+00:00 duration:11.677306
[2024-08-24T12:20:18.798+0000] {logging_mixin.py:190} INFO - Task:<Task(SparkSubmitOperator): spark_submit_task> dag:<DAG: spark_***_dag> dagrun:<DagRun spark_***_dag @ 2024-08-24 11:34:17.052163+00:00: manual__2024-08-24T11:34:17.052163+00:00, state:running, queued_at: 2024-08-24 12:20:05.553906+00:00. externally triggered: True>
[2024-08-24T12:20:18.799+0000] {logging_mixin.py:190} INFO - Failure caused by Cannot execute: spark-submit --master spark://543daafc0f5f:7077 --name arrow-spark --queue root.default --deploy-mode client /opt/***/dags/spark_etl_script_docker.py. Error code is: 1.
[2024-08-24T12:20:18.799+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=spark_***_dag, task_id=spark_submit_task, run_id=manual__2024-08-24T11:34:17.052163+00:00, execution_date=20240824T113417, start_date=20240824T122007, end_date=20240824T122018
[2024-08-24T12:20:18.815+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-24T12:20:18.815+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 68 for task spark_submit_task (Cannot execute: spark-submit --master spark://543daafc0f5f:7077 --name arrow-spark --queue root.default --deploy-mode client /opt/***/dags/spark_etl_script_docker.py. Error code is: 1.; 2589)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 2995, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3149, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3173, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 505, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://543daafc0f5f:7077 --name arrow-spark --queue root.default --deploy-mode client /opt/***/dags/spark_etl_script_docker.py. Error code is: 1.
[2024-08-24T12:20:18.849+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 1
[2024-08-24T12:20:18.864+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-24T12:20:18.867+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
